### PR TITLE
Update NOTICE.txt

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

File: NOTICE.txt [hunk 1]
```diff
@@ -124,6 +124,19 @@ SOFTWARE.
...
 ---
 
+## @mattermost/calls
+
+This product contains '@mattermost/calls' by Mattermost.
+
+Calls enables voice calling and screen sharing functionality in Mattermost channels.
+
+* HOMEPAGE:
+  * https://github.com/mattermost/mattermost-plugin-calls
```
Comment (cpoile) [inline NOTICE.txt:134]:
```suggestion
  * https://github.com/mattermost/calls-common
```

**Review by cpoile — APPROVED** [review]
Probably should point to the repo that's being included. I'm 0/5 for calling it `mattermost/calls-common` rather than `@mattermost/calls` in the `NOTICE.txt`.

Thanks Amy!

**Comment by enahum** [comment]
> Probably should point to the repo that's being included. I'm 0/5 for calling it `mattermost/calls-common` rather than `@mattermost/calls` in the `NOTICE.txt`.
> 
> 
> 
> Thanks Amy!

This means that the package name should also change

**Review by enahum — CHANGES_REQUESTED** [review]

File: NOTICE.txt [hunk 2]
```diff
@@ -124,6 +124,19 @@ SOFTWARE.
...
 ---
 
+## @mattermost/calls
+
+This product contains '@mattermost/calls' by Mattermost.
+
+Calls enables voice calling and screen sharing functionality in Mattermost channels.
+
+* HOMEPAGE:
+  * https://github.com/mattermost/calls-common
```
[the remark this answers is not in the document]
Comment (cpoile) [reply NOTICE.txt:134]: @enahum I'm not sure what you mean? It does (you can try it)

**Review by cpoile — APPROVED** [review]

**Pull request merged**